src/transform/ancestor.js

```diff
@@ -118,7 +118,7 @@ function canBeLifted(doc, range) {
   }
 }
 
-// :: (Node, Pos, ?Pos) → bool
+// :: (Node, Pos, ?Pos) → ?{found: any, range: any}
```

Comment:
You can have a look at my stuff on https://github.com/adrianheine/prosemirror/tree/frain. Just run `frain.sh` in the top dir, it's down to 67 errors for me. It's pretty tedious but also strangely fun to work through the issues one-by-one, but I actually enjoy working in a non-effective way :) Most of the hacking-around happens in `handleBadLines` in `frain/frainparse.js`. `frain/parsePmType.js` is mostly the getdocs parsing, but also some special-casing in `mapType`.